lighthouse-core/gather/computed/metrics/interactive.js

```diff
@@ -90,6 +90,9 @@ class Interactive extends MetricArtifact {
    * @return {{cpuQuietPeriod: TimePeriod, networkQuietPeriod: TimePeriod, cpuQuietPeriods: Array<TimePeriod>, networkQuietPeriods: Array<TimePeriod>}}
    */
   static findOverlappingQuietPeriods(longTasks, networkRecords, traceOfTab) {
+    if (!traceOfTab.timestamps.firstContentfulPaint) {
+      throw new LHError(LHError.errors.NO_FCP);
+    }
```

Comment:
this was already checked down in `computeObservedMetric()`, but this seemed better than passing in the known-defined value as another parameter?